src/sentry/incidents/action_handlers.py

```diff
@@ -154,18 +154,19 @@ def format_duration(minutes):
     """
 
     if minutes >= 1440:
-        days = minutes // 1440
-        return "{} day{}".format(days, pluralize(days))
+        days = int(minutes // 1440)
```

Comment:
```
>>> type(3 // 2)
<type 'int'>
```

This is the case for both in Py2 and Py3 so why to we need the cast?

Reply:
Oh wait, because `minutes` can be a `float`?

Reply:
Yeah

Reply:
Then wouldn't it be safer/easier to do `minutes = int(minutes)` at the top of the function and leave the remaining as they are?